packages/plugins/EVM/src/state/Connection/connection.ts

```diff
@@ -154,10 +154,10 @@ class Connection implements EVM_Connection {
         }
     }
 
-    private getOptions(
+    private getOptions = (
```

Comment:
It's poor maintainability. How can others know whether if need to use the arrow function? Why not use `bind`?